doc/cluster_spec.md

```diff
@@ -27,6 +27,9 @@ Some options in a running cluster specification can be changed to update the des
 | usePgrewind               | try to use pg_rewind for faster instance resyncronization.                                                                                                                                                                                                                                                                                                                                                                                                                        | no                        | bool              | false                                                                                                                               |
+| pgrewindInterval          | interval to wait until next pg_rewind try                                                                                                                                                                                                                                                                                                                                                                                                                                         | no                        | string (duration) | 0s                                                                                                                                  |
+| pgrewindTimeout           | time after which we stop trying to pg_rewind                                                                                                                                                                                                                                                                                                                                                                                                                                      | no                        | string (duration) | 0s                                                                                                                                  |
+| pgrewindCheckpoint        | force checkpoint on primary before performing pg_rewind                                                                                                                                                                                                                                                                                                                                                                                                                           | no                        | bool              | false                                                                                                                               |
```

Comment:
This is named differently to the PR to add this configuration parameter: https://github.com/sorintlab/stolon/pull/644

I think maybe we should leave the removal of this hardcoding out of this PR?